packages-exp/auth-exp/src/core/persistence/browser.test.ts

```diff
@@ -31,7 +36,7 @@ describe('core/persistence/browser', () => {
   afterEach(() => sinon.restore());
 
   describe('browserLocalPersistence', () => {
-    const persistence: Persistence = browserLocalPersistence as Persistence;
+    const persistence: Persistence = (browserLocalPersistence as PersistenceInstantiator)._getInstance();
```

Comment:
you do this cast all over the place, why not make this an importable function like `_getPeristenceInstance(persistence: externs.Persistence): Persistence`

Reply:
Done